### PR TITLE
[ONNX FE Tests] Fix ONNX FE tests on Debian

### PR DESCRIPTION
**Pull request by rkazants**
**Details:** Fixing error "Cannot load library libtest_builtin_extensions_1.so libopenvino_tensorflow_fe.so"

**Ticket:** TBD

Signed-off-by: Kazantsev, Roman <roman.kazantsev@intel.com>

### CODE REVIEW

File: .ci/azure/linux_debian.yml [hunk 1]
```diff
@@ -295,7 +295,9 @@ jobs:
     displayName: 'OV Core UT'
     continueOnError: false
 
-  - script: $(INSTALL_TEST_DIR)/ov_onnx_frontend_tests --gtest_print_time=1 --gtest_filter=-*IE_GPU* --gtest_output=xml:$(INSTALL_TEST_DIR)/TEST-ONNXFrontend.xml
+  - script: |
+      export OV_FRONTEND_PATH=$(INSTALL_TEST_DIR):/usr/lib/x86_64-linux-gnu/
```
Comment (ilya-lavrenov) [inline .ci/azure/linux_debian.yml:299]:
```suggestion
      export LD_LIBRARY_PATH=$(INSTALL_TEST_DIR)
```

**Review by ilya-lavrenov — APPROVED** [review]

File: .ci/azure/linux_debian.yml [hunk 2]
```diff
@@ -295,7 +295,9 @@ jobs:
     displayName: 'OV Core UT'
     continueOnError: false
 
-  - script: $(INSTALL_TEST_DIR)/ov_onnx_frontend_tests --gtest_print_time=1 --gtest_filter=-*IE_GPU* --gtest_output=xml:$(INSTALL_TEST_DIR)/TEST-ONNXFrontend.xml
+  - script: |
+      export LD_LIBRARY_PATH=$(INSTALL_TEST_DIR)
+      $(INSTALL_TEST_DIR)/ov_onnx_frontend_tests --gtest_print_time=1 --gtest_filter=-*IE_GPU* --gtest_output=xml:$(INSTALL_TEST_DIR)/TEST-ONNXFrontend.xml
```
Comment (ilya-lavrenov) [inline .ci/azure/linux_debian.yml:300]: please, copy the same line for PDPD tests

Comment (rkazants) [reply .ci/azure/linux_debian.yml:300]: Done

**Pull request merged**